blockchain/stake/internal/tickettreap/common.go

```diff
@@ -62,8 +62,8 @@ type treapNode struct {
 
 // nodeSize returns the number of bytes the specified node occupies including
 // the struct fields and the contents of the key and value.
-func nodeSize(node *treapNode) uint64 {
-	return nodeFieldsSize + uint64(len(node.key)+nodeValueSize)
+func nodeSize(node *treapNode) int {
```

Comment:
This shouldn't be changed.  It is intentionally not using `int` because `int` varies per architecture and this would thus cause different consensus rules between archs.